apps/web/frontend/components/reportForm/reportForm.tsx

```diff
@@ -123,7 +125,7 @@ export const ReportForm = () => {
               value={selectedProject}
               getLabel={(project) => project.title}
               getKey={(project) => project.id}
-              onChange={(project) => setSelectedProjectId(project?.id)}
+              onChange={(project) => router.push(`/reports/${format(date, 'yyyy-MM')}/${project?.id ?? ''}`)}
```

Comment:
@inpaceee, What happens when you have a user selected and you change the project? The user should still be selected. Does this still works?